src/lib/docs/get-docs-nav-items.ts

```diff
@@ -0,0 +1,45 @@
+import {
+	DocsNavItem,
+	ProductData,
+	ProductSlug,
+	RootDocsPath,
+} from 'types/products'
+
+/**
+ * Given ProductData for a specific product,
+ * Return items to render in a "Documentation" navigation section.
+ */
+export function getDocsNavItems(
+	currentProduct: Pick<ProductData, 'slug' | 'rootDocsPaths' | 'docsNavItems'>
+): DocsNavItem[] {
+	const { slug, docsNavItems, rootDocsPaths } = currentProduct
+	if (!docsNavItems) {
+		return rootDocsPaths.map((r: RootDocsPath) => {
+			return docsNavItemFromRootDocsPath(r, slug)
+		})
+	} else {
+		return docsNavItems.map((entry: DocsNavItem | string) => {
+			if (typeof entry === 'string') {
+				const match = rootDocsPaths.find((r: RootDocsPath) => r.path == entry)
```

Comment:
`docsNavItems` supports string values as a shorthand for existing `rootDocsPaths`, while also supporting custom items (which seem necessary to meet Terraform's use case).